src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectMessage.cs

```diff
@@ -198,7 +198,7 @@ private void EnsureTelemetryValues(OpenIdConnectMessage clonedMessage)
             if (this.EnableTelemetryParameters)
             {
                 clonedMessage.SetParameter(OpenIdConnectParameterNames.SkuTelemetry, SkuTelemetryValue);
-                clonedMessage.SetParameter(OpenIdConnectParameterNames.VersionTelemetry, typeof(OpenIdConnectMessage).GetTypeInfo().Assembly.GetName().Version.ToString());
+                clonedMessage.SetParameter(OpenIdConnectParameterNames.VersionTelemetry, IdentityModelTelemetryUtil.ClientVer);
```

Comment:
sku should be fine to use between both, it is held constant. version, I'm not sure about, generally we expect them to be the same, but with binding redirects for example, is that always true?

Reply:
Reverted the change.